docs/ref/contrib/admin/index.txt

```diff
@@ -1606,6 +1606,15 @@ templates used by the :class:`ModelAdmin` views:
             def get_sortable_by(self, request):
                 return {*self.get_list_display(request)} - {'rank'}
 
+.. method:: ModelAdmin.get_inlines(request, obj=None)
+
+.. versionadded:: 3.0
+
+    The ``get_inlines`` method is given the ``HttpRequest`` and the
+    ``obj`` being edited (or ``None`` on an add form) and is expected to return
+    an iterable of inline objects, You can override this method to dynamically add
+    inlines to a model instead of specifying them in :attr:`ModelAdmin.inlines`.
```

Comment:
`...to dynamically add inlines to a model instead of specifying...` -> 
`...to dynamically add inlines based on the request or model instance instead of specifying...`